### PR TITLE
python-instagram-private-api: add package

### PR DESCRIPTION
**Pull request by D3vil0p3r**
It is a dependency that is required for Osintgram tool: https://github.com/BlackArch/blackarch/pull/3654

### CODE REVIEW

File: packages/python-instagram-private-api/PKGBUILD [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+
+pkgname=python-instagram-private-api
+_pkgname=instagram_private_api
+pkgver=1.6.0.0
+pkgrel=1
+pkgdesc='A Python wrapper for the Instagram private API with no 3rd party dependencies.'
+arch=('any')
+url='https://pypi.org/project/instagram-private-api'
+license=('MIT')
+depends=()
```
Comment (noraj) [inline packages/python-instagram-private-api/PKGBUILD:12]:
```suggestion
depends=('python')
```

File: packages/python-instagram-private-api/PKGBUILD [hunk 2]
```diff
@@ -0,0 +1,28 @@
+# This file is part of BlackArch Linux ( https://www.blackarch.org/ ).
+# See COPYING for license details.
+
+pkgname=python-instagram-private-api
+_pkgname=instagram_private_api
+pkgver=1.6.0.0
+pkgrel=1
+pkgdesc='A Python wrapper for the Instagram private API with no 3rd party dependencies.'
+arch=('any')
+url='https://pypi.org/project/instagram-private-api'
```
Comment (noraj) [inline packages/python-instagram-private-api/PKGBUILD:10]:
```suggestion
url='https://github.com/ping/instagram_private_api'
```

File: packages/python-instagram-private-api/PKGBUILD [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+build() {
+  cd "$_pkgname-$pkgver"
+
+  python setup.py build
+}
+
+package() {
+  cd "$_pkgname-$pkgver"
+
+  python setup.py install --root="$pkgdir" --prefix=/usr -O1 --skip-build
```
Comment (noraj) [inline packages/python-instagram-private-api/PKGBUILD:27]: This is just a comment to tell you that you can test your PKGBUILD in a clean sandbox with `ba-dev` cf. https://gist.github.com/noraj/5045be32490f1161161e16fc5820066c#testing

E.G.

```
$ ba-dev -b
$ ba-dev -e 'python' -p python-instagram-private-api-1.6.0.0-1-any.pkg.tar.zst
Package python-instagram-private-api-1.6.0.0-1-any.pkg.tar.zst installed correctly! Testing it now...
Python 3.10.9 (main, Dec 19 2022, 17:35:49) [GCC 12.2.0] on linux
Type "help", "copyright", "credits" or "license" for more information.
>>> from instagram_private_api import Client, ClientCompatPatch
```

**Review by noraj — CHANGES_REQUESTED** [review]

File: packages/python-instagram-private-api/PKGBUILD (hunk 3 above)
Comment (D3vil0p3r) [reply packages/python-instagram-private-api/PKGBUILD:27]: Does it work only with packages of python tools or with all packages?

Comment (noraj) [reply packages/python-instagram-private-api/PKGBUILD:27]: Testing all packages :)

**Pull request merged**